src/diagram/components/diagram.tsx

```diff
@@ -1,25 +1,24 @@
 import { observer } from "mobx-react-lite";
-import React, { useRef, useState } from "react";
-import ReactFlow, { Edge, Elements, OnConnectFunc, isEdge,
-  OnEdgeUpdateFunc, MiniMap, Controls, ReactFlowProvider, FlowTransform, OnConnectStartFunc, OnConnectEndFunc } from "react-flow-renderer/nocss";
+import React, { useCallback, useRef, useState } from "react";
+import ReactFlow, {
+  Controls, Edge, MiniMap, OnConnectEnd, OnConnectStart,
+  OnEdgeUpdateFunc, OnNodesDelete, ReactFlowProvider, Viewport
+} from "reactflow";
 
 import { DQRootType } from "../models/dq-root";
+import { DQNodeType } from "../models/dq-node";
 import { QuantityNode } from "./quantity-node";
 import { FloatingEdge } from "./floating-edge";
 import { ToolBar } from "./toolbar";
 import { DiagramHelper } from "../utils/diagram-helper";
 import { ConnectionLine } from "./connection-line";
 import { MarkerEnd } from "./marker-end";
 
-// We use the nocss version of RF so we can manually load
-// the CSS. This way we can override it.
-// Otherwise RF injects its CSS after our CSS, so we can't
-// override it.
-import "react-flow-renderer/dist/style.css";
-import "react-flow-renderer/dist/theme-default.css";
+import "reactflow/dist/style.css";
```

Comment:
Sorry, I left this and the following CSS file import here, but re-reading the deleted comment, I'm thinking dropping these two lines may be possible now. Did you try leaving them out?

Reply:
Forget I said the above. I tried removing the lines and doing so definitely breaks things.

It doesn't quite make sense to me that these lines would help if React Flow is already injecting the CSS, but apparently it does. I see they do something similar in [their examples of custom nodes](https://reactflow.dev/docs/guides/custom-nodes/), so looks like the right thing to do.